tests/csm/rest/test_capacity_quota.py

```diff
@@ -1683,6 +1689,616 @@ def test_41970(self):
+        res = self.csm_obj.verify_max_objects(data_size, num_objects, self.akey, self.skey,
+                                              self.bucket)
+        assert res[0], res[1]
+        object_list = s3_misc.get_objects_list(self.bucket,
+                    self.akey, self.skey)
+        self.log.info("Object list is %s ", object_list)
+
+        self.log.info("Step 3: Get capacity count from AWS")
+        total_objects, total_size = s3_misc.get_objects_size_bucket(self.bucket,
+                            self.akey, self.skey)
+        self.log.info("total objects and size %s and %s ", total_objects, total_size)
+        self.log.info("Data size is %s ", data_size)
+
+        self.log.info("Step 4: Perform & Verify GET API to get capacity usage stats")
+        resp = self.csm_obj.get_user_capacity_usage("user", self.user_id)
+        assert resp.status_code == HTTPStatus.OK, \
+                "Status code check failed for get capacity"
+        uid = resp.json()["capacity"]["s3"]["users"][0]["id"]
+        t_obj = resp.json()["capacity"]["s3"]["users"][0]["objects"]
+        t_size = resp.json()["capacity"]["s3"]["users"][0]["used"]
+        m_size = resp.json()["capacity"]["s3"]["users"][0]["used_rounded"]
+
+        assert_utils.assert_equals(self.uid, uid, "id is not equal")
+        assert_utils.assert_equals(total_objects, t_obj, "Number of objects not equal")
+        assert_utils.assert_equal(total_size, t_size, "Total Size mismatch found")
+        assert_utils.assert_greater_equal(m_size, total_size, "Total Used Size mismatch found ")
+        assert_utils.assert_equal(m_size, t_size, "used_rounded and used mismatch found ")
+
+        self.log.info("##### Test ended -  %s #####", test_case_name)
+
+
+    @pytest.mark.lc
+    @pytest.mark.csmrest
+    @pytest.mark.cluster_user_ops
+    @pytest.mark.parallel
+    @pytest.mark.tags('TEST-43837')
+    def test_43837(self):
+        """
+        Verify Max size User level quota & capacity for IAM user with check_on_raw=False
+        with odd multiples of aligned size
+        """
+        test_case_name = cortxlogging.get_frame()
+        self.log.info("##### Test started -  %s #####", test_case_name)
+        num_objects = 1
+        odd_multiplier = self.csm_obj.random_gen.randrange(1, 20, 2)
+        self.log.info("Number of objects to be created are: %s", num_objects)
+        data_size = self.aligned_size * odd_multiplier * 1024
+
+        self.log.info("Step 1: Perform PUT API to set user level quota")
+        payload = self.csm_obj.iam_user_quota_payload(True, data_size, num_objects,
+                                 check_on_raw=False)
+        result, resp = self.csm_obj.verify_get_set_user_quota(self.user_id, payload,
+                                                              verify_response=False)
+        assert result, "Verification for get set user failed."
+        self.log.info("Response : %s", resp)
```

Comment:
This can be removed as this error message

Reply:
done